src/flb_input_chunk.c

```diff
@@ -1451,8 +1478,12 @@ int flb_input_chunk_append_raw(struct flb_input_instance *in,
 const void *flb_input_chunk_flush(struct flb_input_chunk *ic, size_t *size)
 {
     int ret;
+    size_t pre_size, post_size;
```

Comment:
Could you put those in individual lines?

Reply:
done!